inc/Engine/CriticalPath/CriticalCSSSubscriber.php

```diff
@@ -437,10 +435,50 @@ public function insert_critical_css_buffer( $buffer ) {
 		$critical_css_content = str_replace( '\\', '\\\\', $critical_css_content );
 
 		$buffer = preg_replace( '#</title>#iU', '</title><style id="rocket-critical-css">' . wp_strip_all_tags( $critical_css_content ) . '</style>', $buffer, 1 );
+		$buffer = preg_replace( '#</body>#iU', $this->return_remove_cpcss_script() . '</body>', $buffer, 1 );
 
 		return $buffer;
```

Comment:
Return buffer from the `preg_replace`.

```suggestion
		return preg_replace( '#</body>#iU', $this->return_remove_cpcss_script() . '</body>', $buffer, 1 );
```